Lab 2.ipynb

```diff
@@ -0,0 +1,927 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "new_pdf2 = convolv(pdf,pdf,1500) # convolve 1500 times\n",
+    "length = len(new_pdf2)\n",
+    "fig3,ax3 = plt.subplots()\n",
+    "plt.yscale(\"log\")\n",
+    "y = np.linspace(1,length,length)\n",
+    "ax3.plot(y/len(y),new_pdf2,'r',lw=\"3\",label=\"Averaged log plot after 1500 convolutions\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "a16a7b0e-f0f3-411d-8075-6fbfa16f06a4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we can find that as we convolute 1500 days, we can find the pdf becoming more like Gaussian, just like what \n",
+    "# we were expecting from the central limit theorem"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "id": "6163c3a0-b58c-4706-9c91-c362236cb1bc",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.2964235162523031"
+      ]
+     },
+     "execution_count": 35,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# pick Y = 4 >> 1, and N = 50\n",
+    "Y = 4\n",
+    "N = 50\n",
+    "new_pdf = convolv(pdf,pdf,N)\n",
+    "res = np.cumsum(new_pdf)\n",
+    "# I would expect the Y*N is the last element in this array since the max result is Y*N = 200\n",
+    "prob = res[-1]\n",
+    "sigma = stats.norm.ppf(1-prob)\n",
+    "sigma"
```

Comment:
Could consider adding some observations about the result of sigma.

Reply:
Got it